software/contrib/update_firmware.py

```diff
@@ -0,0 +1,34 @@
+from machine import bootloader, reset
+from europi_script import EuroPiScript
+from europi import b1, b2, oled
+from os import remove
+
+
+class UpdateFirmware(EuroPiScript):
+    @classmethod
+    def display_name(cls):
+        """Push this script to the end of the menu."""
+        return "~Update Firmware"
+        
+    def back(self):
+        try:
+            remove('saved_state_BootloaderMenu.txt')	#This file needs to be removed so that the menu is returned to rather than this script once it's entered bootloader mode
```

Comment:
[Discussion] This feels like functionality that `EuroPiScript` should provide as a part of an `exit_script()` method. That would keep all of the details of the saved state files in one place.

Reply:
I agree, maybe exit_script() with an optional parameter of deleting the bootloader saved state would be handy, I can look at that in a separate PR after this one because some other scripts might benefit from it too e.g. if the config menu becomes interactive and menu accessible, an on screen button to exit to the menu (or even a hard reset option) might be nice rather than having to hold both buttons again